mouse/mouse_c.h

```diff
@@ -99,7 +99,7 @@ void moveMouse(MMPointInt32 point){
 		// Mouse motion is now done using SendInput with MOUSEINPUT.
 		// We use Absolute mouse positioning
 		#define MOUSE_COORD_TO_ABS(coord, width_or_height) ( \
-			((65536 * coord) / width_or_height) + (coord < 0 ? -1 : 1))
```

Comment:
I think must the #define and (()) code, and the also not right in the multi screen.